packages/TelegramClient-Core.package/TCCTextMessage.class/instance/asSnippet.st

```diff
@@ -0,0 +1,4 @@
+accessing
```

Comment:
not consistent with `TCCNullMessage`, I prefer `accessing`

Reply:
beware: Other message classes also use `accessing`, so you will have to change those too

Reply:
changed to accessing